code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -294,7 +294,7 @@
 						// you need earmuffs, inacusiate, or replacement
 				else if(ears.ear_damage >= 5)
 					to_chat(src, "<span class='warning'>Your ears start to ring!</span>")
-				src << sound('sound/weapons/flash_ring.ogg',0,1,0,250)

```

Comment:
http://www.byond.com/docs/ref/info.html#/proc/sound

`volume: 100 for full volume (default), 0 for none, or any value in between `